src/KoState.cpp

```diff
@@ -83,9 +65,8 @@ void KoState::play_move(int color, int vertex) {
     if (vertex != FastBoard::PASS && vertex != FastBoard::RESIGN) {
         FastState::play_move(color, vertex);
 
-        m_ko_hash_history.push_back(board.get_ko_hash());
-        m_hash_history.push_back(board.get_hash());
     } else {
-        play_pass();
+        FastState::play_pass();
```

Comment:
This seems a bit weird given that the same patch changes KoState::play_pass. Intentional?

Reply:
intentional, I preferred both play_pass and play_move to call this method, so there's one fewer m_ko_hash_history.push_back.